src/Makefile.test.include

```diff
@@ -167,6 +167,8 @@ check-local: $(BITCOIN_TESTS:.cpp=.cpp.test)
 	$(PYTHON) $(top_builddir)/test/util/bitcoin-util-test.py
 	@echo "Running test/util/rpcauth-test.py..."
 	$(PYTHON) $(top_builddir)/test/util/rpcauth-test.py
+	@echo "Running bench/bench_bitcoin -evals=1 -scaling=0..."
+	$(top_builddir)/src/bench/bench_bitcoin -evals=1 -scaling=0 2> >(grep -vE "^WARNING: ") > /dev/null
```

Comment:
Does scaling=0 actually run everything at least once? If not, just set to a small value larger than zero.

Reply:
@MarcoFalke Yes it does, see:

https://github.com/bitcoin/bitcoin/blob/78dae8caccd82cfbfd76557f1fb7d7557c7b5edb/src/bench/bench.cpp#L115-L118